### PR TITLE
Initial project

### PR DESCRIPTION
**Pull request by AltenGD**
dont merge it yet tho

### CODE REVIEW

File: GDEdit/GDE.App/Program.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using osu.Framework;
+using osu.Framework.Platform;
+using GDE.App.Main;
+
+namespace GDE.App
+{
+    class Program
```
Comment (AlFasGD) [inline GDEdit/GDE.App/Program.cs]: Make it `public static class` instead

File: GDEdit/GDE.App/Program.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    {
+        static void Main(string[] args)
```
Comment (AlFasGD) [inline GDEdit/GDE.App/Program.cs]: `public static void`

File: GDEdit/GDE.Tests/Program.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using osu.Framework;
+using osu.Framework.Platform;
+
+namespace GDE.Tests
+{
+    class Program
```
Comment (AlFasGD) [inline GDEdit/GDE.Tests/Program.cs]: `public static class`

File: GDEdit/GDE.Tests/Program.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    {
+        [STAThread]
+        public void Main(string[] args)
```
Comment (AlFasGD) [inline GDEdit/GDE.Tests/Program.cs]: `public static void`

**Comment by AltenGD** [comment]
ok ready to merge now :ok_hand:

**Pull request merged**